packages/perseus/src/components/__stories__/multi-button-group.stories.tsx

```diff
@@ -2,65 +2,47 @@ import * as React from "react";
 
 import MultiButtonGroup from "../multi-button-group";
 
-type StoryArgs = {
-    allowEmpty: boolean;
-};
-
-type Story = {
-    title: string;
-    args: StoryArgs;
-};
+import type {PropsFor} from "@khanacademy/wonder-blocks-core";
+import type {Meta, StoryObj} from "@storybook/react";
 
-export default {
+const meta: Meta = {
     title: "Perseus/Components/Muli-Button Group",
+    component: MultiButtonGroup,
     args: {
         allowEmpty: true,
+        buttons: [],
+    },
+    render: function WithState(props: PropsFor<typeof MultiButtonGroup>) {
+        const [values, updateValues] = React.useState(props.values);
+        return (
+            <MultiButtonGroup
+                {...props}
+                values={values}
+                onChange={updateValues}
+            />
+        );
     },
-} as Story;
-
-const HarnassedButtonGroup = (
-    props: Pick<
-        React.ComponentProps<typeof MultiButtonGroup>,
-        "buttons" | "allowEmpty"
-    >,
-) => {
-    const [values, updateValues] = React.useState(
-        null as ReadonlyArray<string> | null | undefined,
-    );
-
-    return (
-        <MultiButtonGroup
-            {...props}
-            values={values}
-            onChange={(newValues) => {
-                updateValues(newValues);
-            }}
-        />
-    );
 };
+export default meta;
 
-export const ButtonsWithNoTitles = (args: StoryArgs): React.ReactElement => {
-    return (
-        <HarnassedButtonGroup
-            {...args}
-            buttons={[
-                {value: "One", content: "Item #1"},
-                {value: "Two", content: "Item #2"},
-                {value: "Three", content: "Item #3"},
-            ]}
-        />
-    );
+type Story = StoryObj<typeof MultiButtonGroup>;
+
+export const ButtonsWithNoTitles: Story = {
+    args: {
+        buttons: [
+            {value: "One", content: "Item #1"},
+            {value: "Two", content: "Item #2"},
+            {value: "Three", content: "Item #3"},
+        ],
+    },
 };
 
```

Comment:
**Optional:** This could be a good opportunity to add a story description using JS Docs right above the story definition.

```
/** 
 * Hover over the buttons to see their titles.
 */
```

Reply:
![giphy](https://github.com/user-attachments/assets/0a158d10-9660-4017-8dd9-c24d86cac5bf)